docs/pages/get-started.mdx

```diff
@@ -39,15 +38,25 @@ end
 
 subgraph server[Linux server]
     direction RL
-    auth["Auth Service"]
     proxy[Proxy Service]
+    auth["Auth Service"]
     ssh[SSH Service]
-    auth-- "Short-lived credentials" -->proxy
-    ssh-- Reverse tunnel-->proxy
+    proxy<-- "Short-lived credentials" -->auth
+    proxy<-- Reverse tunnel-->ssh
 end
 
-Developers-- tsh sessions --->server
-serv1 & serv2 & serv3-- Reverse tunnel--->server
+Developers-- tsh sessions ---->proxy
+serv1 & serv2 & serv3<-- Reverse tunnel---->proxy
+
+classDef teleportColor fill:#844cff,color:#fff
```

Comment:
Same feedback as above re: the use of color. The "Teleport service" coloration also seems pretty low-contrast to me.

Reply:
Standardized to a single `teleportColor`.